apps/launch/common-util/constants/urls.ts

```diff
@@ -0,0 +1,19 @@
+import { arbitrum, base, celo, gnosis, mainnet, optimism, polygon } from 'viem/chains';
+
+export const GET_VEOLAS_URL = 'https://member.olas.network/';
+
+export const EXPLORE_URLS: Record<string, string> = {
+  1: mainnet.blockExplorers.default.url,
+  10: optimism.blockExplorers.default.url,
+  100: gnosis.blockExplorers.default.url,
+  137: polygon.blockExplorers.default.url,
+  8453: base.blockExplorers.default.url,
+  42_161: arbitrum.blockExplorers.default.url,
+  42_220: celo.blockExplorers.default.url,
+};
+
+// all the urls used in the app
+export const URL = {
+  myStackingContract: 'my-staking-contracts',
```

Comment:
```suggestion
  myStakingContract: 'my-staking-contracts',
```
likely need to refactor where it's imported too :/